src/jdk.crypto.cryptoki/share/classes/sun/security/pkcs11/Config.java

```diff
@@ -500,9 +513,15 @@ private void parse() throws IOException {
             case "nssOptimizeSpace"->
                 nssOptimizeSpace = parseBooleanEntry(st.sval);
             default->
-                throw new ConfigurationException
-                        ("Unknown keyword '" + st.sval + "', line " +
-                        st.lineno());
+                {
+                    if ("tokenLabel".equalsIgnoreCase(st.sval)) {
+                        tokenLabel = parseStringEntry("tokenLabel");
```

Comment:
This needs to be:
```java
                        st.sval = "tokenLabel";
                        tokenLabel = parseStringEntry(st.sval);
```
See https://github.com/ibmruntimes/openj9-openjdk-jdk/pull/547/files#r1104771960.